go.mod

```diff
@@ -27,9 +34,10 @@ require (
 	github.com/prometheus/prometheus v1.8.2-0.20200507164740-ecee9c8abfd1
 	github.com/spf13/cobra v0.0.7 // indirect
 	github.com/testcontainers/testcontainers-go v0.5.1
+	github.com/uber/jaeger-client-go v2.23.0+incompatible
```

Comment:
as near as I can tell, all of the things here are either the same as in the old version, a removal, or an indirect. Would you try to remove the new lines and see if it still works? It seems odd that all of this should be needed